### PR TITLE
show indicator for users who have mastered a set in the Most Points Earned list

### PR DESCRIPTION
**Pull request by Jamiras**
https://discord.com/channels/310192285306454017/1276228879761281124/1276676118115647530

image

### CODE REVIEW

File: resources/views/components/game/top-achievers/score-row.blade.php [hunk 1]
```diff
@@ -15,5 +16,12 @@
             <x-user.avatar display="icon" icon-size="xs" :user="$user" />
             <x-user.avatar hasHref="true" :user="$user" />
         </td>
-        <td class="text-right">{{ localized_number($score) }}</td>
+        <td>
+            <div class="flex items-center gap-1 float-right">
```
Comment (wescopeland) [inline resources/views/components/game/top-achievers/score-row.blade.php:20]: With a flex layout, "justify-end" is preferable to "float-right".

This doc is a great explainer on flex justify with helpful infographics: https://tailwindcss.com/docs/justify-content

File: resources/views/components/game/top-achievers/score-row.blade.php [hunk 2]
```diff
@@ -15,5 +16,12 @@
...
+                @if ($score === $maxScore)
+                    <div class="rounded-full bg-[gold] light:bg-yellow-600 w-2 h-2"></div>
```
Comment (wescopeland) [inline resources/views/components/game/top-achievers/score-row.blade.php:22]: Recommend adding a `title="Mastered"` attribute.

In other areas, there's contextual information explaining what the dot means (either a label or a legend elsewhere). For accessibility, we probably need something here too.

**Review by wescopeland — APPROVED** [review]

**Comment by wescopeland** [comment]
Not sure if you'd like to also fix while in here, I noticed that rows for Most Points Earned have the issue where the avatar component isn't rendering a tooltip for the user.

**Comment by Jamiras** [comment]
> Not sure if you'd like to also fix while in here, I noticed that rows for Most Points Earned have the issue where the avatar component isn't rendering a tooltip for the user.

That's a general problem with using `<x-user.avatar`. I've replaced it with `{!! userAvatar(...`

File: resources/views/components/game/top-achievers/score-row.blade.php [hunk 3]
```diff
@@ -12,8 +13,16 @@
 
         <td class="text-right">{{ $rank }}</td>
         <td class="whitespace-nowrap">
-            <x-user.avatar display="icon" icon-size="xs" :user="$user" />
-            <x-user.avatar hasHref="true" :user="$user" />
+            {!! userAvatar($user, iconSize:16) !!}
+            <!-- <x-user.avatar display="icon" icon-size="xs" :user="$user" /> -->
+            <!-- <x-user.avatar hasHref="true" :user="$user" /> -->
```
Comment (wescopeland) [inline resources/views/components/game/top-achievers/score-row.blade.php:18]: If we want to leave these in, recommend replacing them with `{{-- --}}` style comments.

`<!-- -->` gets shipped to the browser (marginally increasing the document size), whereas the Blade rendering engine strips `{{-- --}}`.

**Comment by wescopeland** [comment]
Yea, that Blade component is not unfortunately not hooked up with the Alpine.js stuff to connect the tooltip cards. Probably not a huge deal if the front-end is moving towards using Inertia.js anyway.

**Review by wescopeland — APPROVED** [review]

**Pull request merged**